portfoliostylesheet.css

```diff
@@ -89,14 +89,18 @@ body {
 	flex-direction:column;
 	wrap:nowrap;
 }
-.careerProfile{display: flex;
+.careerProfile {
+	display: flex;
 	flex-direction:column;
 	align-items:left;
 	wrap:nowrap;
 	margin: 15px;
-	padding: 25px;}
+	padding: 25px;
+}
 	
-.profileBody{}
+.profileBody {
+	
+}
 	
 .projects{
```

Comment:
From line 104 to end the selectors have been indented too much. Can't actually make comments on these lines because of it.